src/tests/test_utils/common_test_utils/include/common_test_utils/data_utils.hpp

```diff
@@ -194,6 +194,60 @@ void inline fill_data_random(T* pointer,
     }
 }
 
+template <>
+void inline fill_data_random(float* pointer,
+                             std::size_t size,
+                             const uint32_t range,
+                             double_t start_from,
+                             const int32_t k,
+                             const int seed) {
```

Comment:
better to move implementations / template specializations to .cpp files

Reply:
> better to move implementations / template specializations to .cpp files

@ilya-lavrenov  Yes, found the implementations in cpp by ilya C. Refacted.